### DISCUSSION TITLE
Scaling a spring vrm

### DISCUSSION THREAD

**Discussion by Samsy** [pixiv/three-vrm#1215] [General]
Hello, 

I'm loading a VRM that includes hair / springs etc..

Once loaded, changing the scale of the scene does not update the spring base root position system, spring physics is then not working as expected

Is there anything we can call to notify the change of scale to the springmanager ?

**Comment by ButzYung** [comment]
I encountered this problem before. This doc should help.

https://github.com/pixiv/three-vrm/blob/dev/docs/spring-bones-on-scaled-models.md

**Comment by Samsy** [comment]
This helped me indeed ! 

Thanks !
